### PR TITLE
Add iNode Energy Meter support

### PR DESCRIPTION
**Pull request by nemoo2580**
[iNode Energy Meter](https://inode.pl/iNode-Energy-Meter-opto,p,27/s_lang/en#) is a wireless energy monitor manufactured by a Polish company ELSAT.

iNode EM bluetooth advertisement is identified by the first two bytes of manufacturerdata (9082, 9282, 9482, 9682 - 4 values, because bit 1 and 2 have an aditional special meaning, not needed for this integration).

Sensor parameters:
- iNodeEM-power - average power consumed in a minute
- iNodeEM-energy - total energy from electricity meter
- iNodeEM-batt - battery percentage

### CODE REVIEW

**Comment by 1technophile** [comment]
I think this device covers the need of #208

**Comment by 1technophile** [comment]
Hi @nemoo2580,

Do you would like to rebase your PR or do you prefer that I do it?

File: docs/prerequisites/devices.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ OpenMQTTGateway is able to scan all the BLE devices that advertise their data so
 | XIAOMI Mi Scale v1 *|XMTZC04HM|weight|
 | XIAOMI Mi Scale v2 *|XMTZC05HM|weight|
 | XIAOMI Mi band * ||steps|
```
Comment (nemoo2580) [inline docs/prerequisites/devices.md:37]: Why Xiaomi Mi band is not supported by HM10 (asterisk)? I don't have HM10 to test my integration, but if it reads advertisement like ESP32, it should work.

Comment (1technophile) [reply docs/prerequisites/devices.md:37]: The devices that rely on `manufacturerdata` are not supported by HM10, so the Mi band and iNodeEM are not currently supported by HM10.
The gateway extract only `servicedata` with HM10

Comment (nemoo2580) [reply docs/prerequisites/devices.md:37]: Ok. I will correct docs.

**Comment by 1technophile** [comment]
Thanks!

**Comment by 1technophile** [comment]
@nemoo2580 could you share some data samples from these devices (manufacturerdata and corresponding measure) so that we can integrate this into some unit tests?

**Comment by nemoo2580** [comment]
> @nemoo2580 could you share some data samples from these devices (manufacturerdata and corresponding measure) so that we can integrate this into some unit tests?

I don't have this device at the moment, but I have found some test data:
`"manufacturerdata": "90826300f0cf0000c409a20080", "battery": 80, "power": 2376, "energy": 21.2928`
`"manufacturerdata": "9082dd0061b80000c409a00080", "battery": 80, "power": 5304.00, "energy": 18.88`

**Pull request merged**